commands/src/main/java/org/wildfly/extras/creaper/commands/ra/RemoveResourceAdapter.java

```diff
@@ -0,0 +1,41 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * Remove existing resource adapter
+ */
+public class RemoveResourceAdapter implements OnlineCommand {
+    private String resourceAdapterId;
+
+    private RemoveResourceAdapter(Builder builder) {
+        this.resourceAdapterId = builder.resourceAdapterId;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+        Operations ops = new Operations(ctx.client);
+        Address address = Address.subsystem("resource-adapters").and("resource-adapter", resourceAdapterId);
+        ops.remove(address);
+    }
+
+    public static final class Builder {
+        private String resourceAdapterId;
```

Comment:
In this exact case, when there's just one parameter, builder is not necessary, the command can have a single-parameter constructor itself. There are commands like that already.